incubator/jenkins-operator/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: 0.1.7
+description: A Helm chart for Kubernetes Jenkins Operator
+name: jenkins-operator
+version: 0.1.7
```

Comment:
The `version` here is the version of the chart which is different from the App as seen in the `appVersion`. Would this be more appropriate to be `0.1.0`? Chart changes can move at a different rate than the applications they deploy.

Reply:
Agreed, however, In the case of this chart we are versioning the operator binary and chart in lockstep. The kubebuilder-based operator build actually generates parts of the chart (RBAC and CRDs)